scss/modules/_introjs.scss

```diff
@@ -0,0 +1,501 @@
+// CSS to support Intro.js
```

Comment:
What about just adding a line explaining that these are vendor styles from introjs?